client/signup/step-wrapper/style.scss

```diff
@@ -0,0 +1,16 @@
+.step-wrapper {
+	.step-wrapper_buttons {
```

Comment:
Super minor, but:

- I don't think you need this extra class. Selecting with `.step-wrapper .skip-step` should be sufficient.
- If you do add it, it should contain two underscores, like `.step-wrapper__buttons`.